lsp-mode.el

```diff
@@ -3354,8 +3357,10 @@ in that particular folder."
                                   (error nil)))
    (when (featurep 'project)
      (when-let ((project (project-current)))
-       (car (with-no-warnings
-              (project-roots project)))))))
+       (if (fboundp 'project-roots)
```

Comment:
You should check for `(fboundp 'project-root)` since that's newer function

Reply:
sure